eks-worker-al2.json

```diff
@@ -45,10 +48,14 @@
           "creator": "{{user `creator`}}"
       },
       "tags": {
-          "created": "{{timestamp}}"
+          "created": "{{timestamp}}",
```

Comment:
I'm fine adding these tags, but do be aware that tags are not visible on public AMIs.